File: ReviewsJuegos/AgregarGenero.vb
Imports System.Data.SqlClient

Public Class AgregarGenero
    Private Sub BtnAgregar_Click(sender As Object, e As EventArgs) Handles BtnAgregar.Click
        ' Crear parámetros para el procedimiento almacenado
        Dim parametros(8) As SqlParameter
        parametros(0) = New SqlParameter("@Genero", TxtAgGenero.Text)
        parametros(1) = New SqlParameter("@IdUsuarioCrea ", TxtAgGenero.Text)




        ' Ejecutar el procedimiento almacenado
        ConexionSQL.ExecuteStoredProcedure("SP_AddGenero", parametros)
        MsgBox("Registro exitoso")
    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        Me.Close()
    End Sub
End Class

File: ReviewsJuegos/AgregarGenero.cs
using System;
using System.Data.SqlClient;
using Microsoft.VisualBasic;

namespace ReviewsJuegos
{

    public partial class AgregarGenero
    {
        public AgregarGenero()
        {
            InitializeComponent();
        }
        private void BtnAgregar_Click(object sender, EventArgs e)
        {
            // Crear parámetros para el procedimiento almacenado
            int IdUsuario = 1;
            var parametros = new SqlParameter[2];
            parametros[0] = new SqlParameter("@Genero", TxtAgGenero.Text);
            parametros[1] = new SqlParameter("@IdUsuarioCrea ", IdUsuario);




            // Ejecutar el procedimiento almacenado
            ConexionSQL.ExecuteStoredProcedure("SP_AgregarGenero", parametros);
            Interaction.MsgBox("Registro exitoso");
        }

        private void Button2_Click(object sender, EventArgs e)
        {
            Close();
        }

        private void TxtAgGenero_TextChanged(object sender, EventArgs e)
        {

        }
    }
}